File: Assignments/Assignment_13/p9.c
#include<stdio.h>
void main(){

        int row;
        printf("Enter the no of rows : \n");
        scanf("%d",&row);
        for(int i=row;i>0;i--){
                int alp=64+row;
                for(int j=1;j<=i;j++){
                        printf("%c\t",alp);
                        alp--;
                }printf("\n\n");
        }
}

File: JAVA/11.Triangle_Pattern/p9.java
import java.util.*;

class Demo{

	public static void main(String[] args){
	
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter no. of rows: ");
		int row = sc.nextInt();
	
		for(int i=row;i>0;i--){
		
			char ch=(char)(64+row);
			for(int j=0;j<i;j++){
			
					System.out.print(ch+"\t");
					ch--;
			}System.out.println();
		}
	}
}
